ui/admin/app/routes/scopes/scope/auth-methods/auth-method/accounts/account.js

```diff
@@ -19,7 +21,20 @@ export default class ScopesScopeAuthMethodsAuthMethodAccountsAccountRoute extend
    * @param {string} params.account_id
    * @return {AccountModel}
    */
-  model({ account_id }) {
+  async model({ account_id }) {
```

Comment:
We are declaring the function async but we aren't `await` for anything. I do understand the we are using `reload: true`, is `async` need here?